zerver/data_import/slack.py

```diff
@@ -1054,7 +1054,7 @@ def get_slack_api_data(slack_api_url: str, get_param: str, **kwargs: Any) -> Any
 
     if data.status_code == requests.codes.ok:
         if 'error' in data.json():
-            raise Exception('Enter a valid token!')
+            raise Exception('Invalid Authorization - You must have a legacy token for your Workspace - see https://api.slack.com/custom-integrations/legacy-tokens')
```

Comment:
This error message can be interpreted in 2 ways:
1. invalid authorization because the user uses legacy token
2. invalid authorization because the user doesn't use legacy token

The message needs to be updated to avoid ambiguity.